engine/src/assets/assets.cpp

```diff
@@ -90,6 +90,57 @@ void Assets::cleanup()
     }
 }
 
+void Assets::importAll(std::string_view path)
+{
+    auto file = core::data::FileSystem::find(path);
+    if (file == nullptr)
+    {
+        CUBOS_ERROR("Couldn't load asset metadata: file {} not found", path);
+        return;
+    }
+
+    if (file->directory())
+    {
+        auto child = file->child();
+        while (child != nullptr)
+        {
+            this->importAll(child->path());
+            child = child->sibling();
+        }
+    }
+    // Skip files that already have a .meta extension.
```

Comment:
```suggestion
    // Skip meta files.
```
I got confused at first, because I'd interpreted this as meaning files for which the meta file exists.